src/service/iot_config.proto

```diff
@@ -378,19 +386,36 @@ service route {
   rpc get(route_get_req_v1) returns (route_v1);
   // Create Route for an Org (auth delegate_keys/owner/admin)
   rpc create(route_create_req_v1) returns (route_v1);
+
   // Update Route for an Org (auth delegate_keys/owner/admin)
   rpc update(route_update_req_v1) returns (route_v1);
   // Delete Route for an Org (auth delegate_keys/owner/admin)
   rpc delete (route_delete_req_v1) returns (route_v1);
-  // Add / Remove / Update EUIs from a Route (auth delegate_keys/owner/admin)
-  rpc euis(route_euis_req_v1) returns (route_euis_res_v1);
-  // Add / Remove/ Update Devaddr Ranges from a Route (auth
+
+  // Get EUIs for a Route (auth delegate_keys/owner/admin)
+  rpc get_euis(route_get_euis_req_v1) returns (stream eui_pair_v1);
```

Comment:
Is there a way other than comments we can communicate the intention of these endpoints (euis/devaddrs) is to receive all the info for a single route then the server will close the connection?

It might just be me, but seeing a stream returned makes me assume subscription.

Reply:
i think proto is too limiting to do much here; we could name long-lived streams `_subscription` or similar to differentiate from a large req/res with a finite limit